core/src/main/java/io/aiven/klaw/service/UtilControllerService.java

```diff
@@ -313,6 +313,7 @@ public AuthenticationInfo getAuth() {
       Map<String, String> teamTopics =
           reqsHandle.getDashboardInfo(commonUtilsService.getTeamId(userName), tenantId);
       authenticationInfo.setMyteamtopics(teamTopics.get("myteamtopics"));
+      authenticationInfo.setMyOrgTopics(teamTopics.get("myOrgTopics"));
```

Comment:
I guess not for this particular PR but it looks like we should split out myTeamTopics and MyOrgTopics into a different stats endpoint.